packages/shared/src/lib/base64.ts

```diff
@@ -0,0 +1,8 @@
+export async function base64ToFile(
+  dataUrl: string,
+  fileName: string,
+): Promise<File> {
+  const res: Response = await fetch(dataUrl);
+  const blob: Blob = await res.blob();
+  return new File([blob], fileName, { type: 'image/png' });
```

Comment:
Why is the image type hard coded here? Is it always `png`?

Reply:
Great catch, extracted the type from the dataurl